cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,293 @@
+# Table of contents: 
```

Comment:
What does line 2 mean ? **Dynamic Objects (Only Supported in JS Objects)** ??

Reply:
`Dynamic Objects (Only Supported in JS Objects)` is a title and moved the title to top and removed `Dynamic Objects (Only Supported in JS Objects)` from TOC